setup.py

```diff
@@ -67,7 +67,7 @@ def run_tests(self):
     "imageio>=2.0,<2.5; python_version<'3.4'",
     "imageio_ffmpeg>=0.2.0; python_version>='3.4'",
     'tqdm>=4.11.2,<5.0',
-    "numpy>=1.17.3; python_version!='2.7'",
+    "numpy>=1.16.6; python_version!='2.7'",
```

Comment:
Do we really need this? how does it works before the changes?

Reply:
the version of numpy we use in py3 in singular is 1.16.6
so I didn't want that by installing it, it would install a newer numpy version...